cogs/message_link.py

```diff
@@ -0,0 +1,60 @@
+import discord
+import re
+from textwrap import shorten
+from discord.ext import commands
+
+
+class MessageLink(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def create_embed(author: discord.User, message: discord.Message):
```

Comment:
This could maybe moved into the JumpView that I proposed in the other comment, which makes the send a bit cleaner in my opinion, also then it can use the `self.message` of the JumpView class and we save a parameter.

instead of
```py
await ctx.send(embed=await MessageLink.create_embed(ctx.author, link), view=jump_view)
```
it then could be 
```py
await ctx.send(embed=jump_view.create_embed(ctx.author), view=jump_view)
```

you would need to rewrite every instance of `message` to `self.message` inside the function though.

I am also fine with how it is now this is just a suggestion.